desktop/cmp/panel/impl/PanelHeader.ts

```diff
@@ -48,9 +49,7 @@ export const panelHeader = hoistCmp.factory({
                         ? box({
                               className: titleCls,
                               flex: 1,
-                              item: isString(title)
-                                  ? span({className: `${titleCls}__inner`, item: title})
-                                  : title
+                              item: span({className: `${titleCls}__inner`, item: title})
```

Comment:
Think this needs to be `collapsed ? collapsedTitle : title` for the vertical collapsed case